presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestPrestoNativeTpchConnectorQueries.java

```diff
@@ -13,20 +13,29 @@
  */
 package com.facebook.presto.nativeworker;
 
+import com.facebook.presto.Session;
 import com.facebook.presto.testing.ExpectedQueryRunner;
 import com.facebook.presto.testing.QueryRunner;
 
 public class TestPrestoNativeTpchConnectorQueries
         extends AbstractTestNativeTpchConnectorQueries
 {
     @Override
-    protected QueryRunner createQueryRunner() throws Exception
+    public Session getSession()
+    {
+        return Session.builder(super.getSession()).setCatalog("tpchstandard").setSchema("tiny").build();
```

Comment:
This code seems the same between Prestissimo and Presto-on-Spark tests. It would be nice to avoid duplicating it.